modules/planning/navi/common/local_path.h

```diff
@@ -0,0 +1,107 @@
+/******************************************************************************
+ * Copyright 2018 The Apollo Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *****************************************************************************/
+
+/**
+ * @file
+ * @brief This file provides the declaration of the class "LocalPath".
+ */
+
+#ifndef MODULES_PLANNING_NAVI_LOCAL_PATH_H_
+#define MODULES_PLANNING_NAVI_LOCAL_PATH_H_
+
+#include <memory>
+#include <vector>
+
+#include "modules/common/math/vec2d.h"
+#include "modules/common/proto/pnc_point.pb.h"
```

Comment:
it is recommended to put system generated headers before user written headers, so a better way is:
```
#include <memory>
#include <vector>

#include "modules/common/proto/pnc_point.pb.h"

#include "modules/common/math/vec2d.h"
```